### PR TITLE
Burn subtitles

### PR DESCRIPTION
**Pull request by howardbaik**
#### Purpose/implementation Section

I wanted to add subtitles to Loqui so I experimented with `ari_burn_subtitles()`and fixed a bug. 

#### What changes are being implemented in this Pull Request?

The `system()` command specified the wrong path to the video output. It was specifying the same path for both the input and the output, which ffmpeg didn't like. I changed the path to the video output so we can have different paths for the input and output.

### CODE REVIEW

**Comment by howardbaek** [comment]
Not sure what the conflict is. Github is showing me a conflict in the NAMESPACE https://github.com/jhudsl/ari/pull/52/conflicts

**Comment by seankross** [comment]
Hey Howard, is there any reason we shouldn't be comparing this PR against the main branch?

**Comment by howardbaik** [comment]
@seankross Technically, this can be compared against the main branch. I'm comparing it against the `ariExtra-immigration` branch because [Loqui](https://loqui.fredhutch.org/) uses ari functions in that branch and not the main branch.

File: R/ari_burn_subtitles.R [hunk 1]
```diff
@@ -4,20 +4,24 @@
...
+#' @param srt Path to subtitle file in \code{srt} format
+#' @param output_video Path to video with subtitles
 #' @param verbose print diagnostic messages.  If > 1,
 #' then more are printed
 #'
 #' @return Name of output video
-ari_burn_subtitles <- function(video, srt, verbose = FALSE) {
+#' @export
+ari_burn_subtitles <- function(input_video, srt,
+                               output_video = tempfile(fileext = ".mp4"),
```
Comment (seankross) [inline R/ari_burn_subtitles.R]: output_video should not have a default value

**Pull request merged**